packages/destination-actions/src/destinations/clevertap/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import userUpload from './userUpload'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Clevertap( Actions)',
+  slug: 'clevertap',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      clevertapAccountId: {
+        label: 'X-CleverTap-Account-Id: ACCOUNT_ID',
```

Comment:
This should probably be more human-friendly since this is the form field label that will be displayed.

Maybe this?
```suggestion
        label: 'CleverTap Account ID',
```